src/fragments/lib-v1/storage/js/remove.mdx

```diff
@@ -1,4 +1,5 @@
-Delete stored data from the storage bucket.
+Delete stored data from the storage bucket. To remove a file, you need to provide the key. `accessLevel` can be 'guest', 'private', or 'protected', defaulting to 'guest'.
+
```

Comment:
We can also add a call out saying that we can only remove from current user and can not `remove` to other users

Reply:
I think that's a good point. Thanks will update